utils/validator.util.js

```diff
@@ -94,6 +94,51 @@ class Validator {
       inStock: Joi.number().positive(),
     }),
   });
+
+  // Validators for review routes
+  static postReviewValidator = celebrate({
+    [Segments.BODY]: Joi.object().keys({
+      reviewText: Joi.string().required(),
+      stars: Joi.number().max(5).min(1),
+      user: Joi.string().required().custom((value, helper) => {
+        if (!isValidObjectId(value)) return helper.message('Please enter a valid user ID');
+        return value;
+      }, 'ObjectID Validation'),
+      book: Joi.string().required().custom((value, helper) => {
+        if (!isValidObjectId(value)) return helper.message('Please enter a valid book ID');
+        return value;
+      }, 'ObjectID Validation'),
+      likes: Joi.array().items(Joi.string().required().custom((value, helper) => {
+        if (!isValidObjectId(value)) return helper.message('Please enter a valid user ID');
+        return value;
+      }, 'ObjectID Validation')),
```

Comment:
Do you want to register review likes at the same time of creating reviews 🤔? But I think review likes should be through another API call that is after the review might have been created.

Reply:
Actually during testing I never actually put in the likes during creation but I just added it in the validation, because it is possible to add the likes

Reply:
I'll remove it